watchfaces/009-funky-town-words.qml

```diff
@@ -29,50 +32,50 @@
 
 import QtQuick 2.1
 
-Canvas {
-    id: rootitem
-    anchors.fill: parent
-    renderTarget: Canvas.FramebufferObject 
+Item {
 
-    onPaint: {
-        var ctx = getContext("2d")
-        ctx.reset()
-        ctx.fillStyle = "white"
-        ctx.textAlign = "center"
-        ctx.textBaseline = 'middle';
-        ctx.shadowColor = "black"
-        ctx.shadowOffsetX = 0
-        ctx.shadowOffsetY = 0
-        ctx.shadowBlur = 3
-
-        var medium = "57 "
-        var thin = "0 "
-        var light = "25 "
-
-        var px = "px "
-
-        var centerX = width/2
-        var centerY = height/2
-
-        var text;
-        if(use12H.value) text = wallClock.time.toLocaleString(Qt.locale(), "hh:mm ap")
-        else             text = wallClock.time.toLocaleString(Qt.locale(), "hh:mm")
-
-        var fontSize = height*0.17
-        var verticalOffset = height*0.025
-        var fontFamily = "Orbitron"
-        ctx.font = medium + fontSize + px + fontFamily;
-        ctx.fillText(text, centerX, centerY+verticalOffset);
+    Image {
+        z: 0
+        width: parent.width
+        height: parent.height
+        source: "img/" + wallClock.time.toLocaleString(Qt.locale(), "hh am").slice(0, 2) + ".svg"
+        anchors {
+            horizontalCenter: parent.horizontalCenter
+            verticalCenter: parent.verticalCenter
+        }
     }
 
-    Connections {
-        target: wallClock
-        onTimeChanged: rootitem.requestPaint()
+    Image {
+        z: 0
+        visible: use12H.value
+        width: parent.width
+        height: parent.height
+        source: "img/" + wallClock.time.toLocaleString(Qt.locale("en_EN"), "ap").toLowerCase().slice(0, 2) + ".svg"
```

Comment:
I know I said I wanted those images in /usr/share/asteroid-launcher/watchfaces/img/ before but I now realize it causes some warnings in asteroid-settings.
Could you please move those to /usr/share/asteroid-launcher/watchfaces-img/ instead? It should be a minor change to the top level asteroid-launcher.pro with a new install target (similar to lines 47, 48)